src/book.py

```diff
@@ -125,6 +141,9 @@ def _read_binance(self, file_path: Path) -> None:
                     operation, utc_time, platform, change, coin, row, file_path
                 )
 
+    def _read_binance_new(self, file_path: Path) -> None:
```

Comment:
Don't name it new. Someday it will change again and we won't name the next one newer :D

Maybe read binance 2. And it's version 2?

Reply:
i dont know if binance has a versioning scheme